### DISCUSSION TITLE
COS存储防盗链设置

### DISCUSSION THREAD

**Discussion by Wink541** [AlistGo/alist#5687] [Q&A]
COS设置防盗链为了防止资源泄露，使用本地代理会被带宽限制导致下载速度慢，防盗链匹配的是referer头，能在请求当中添加一个referer字段吗
![image](https://github.com/alist-org/alist/assets/89731663/ae2a0915-890c-40ad-b8e7-4f12553aef59)

**Comment by linepro6** [comment]
COS 设置私有读写，然后 AList 存储配置启用签名就可以满足你的需求了
